### DISCUSSION TITLE
Errors in Server Log with Little Context

### DISCUSSION THREAD

**Discussion by janjovellanos** [nextgenhealthcare/connect#6356] [Q&A]
Hello Community, we are using Mirth Connect in my organization and we've come across multiple errors populating the server log this morning --

```
ERROR (com.mirth.connect.model.converters.ObjectJSONSeri alizer:38): javax.xml.transform.TransformerException: [com.ctc.wstx.exc.WstxLazyException] com.ctc.wstx.exc.WstxParsingException: Illegal character entity: expansion character (code 0x11
at [row,col {unknown-source}]: [259,209]
```

As you can see -- the error is related to parsing, but I have no context of where or what channel is experiencing the issue. We have many channels and messages flowing at all times. Is there a way to narrow the origin of these types of errors?

Thanks.

**Comment by jonbartels** [comment]
First - please EDIT your post and use code formatting for the error message.

Check `mirth.log`. The error you posted looks like it comes from the MC admin tool. Sometimes the `mirth.log` file on the server has more details.

`ObjectJSONSerializer` so its a channel dealing with JSON. Then somewhere in that message there is a character with hex code `0x11` in ASCII this is a "VT (Vertical Tab)". You'll need to find an example message, load it into an editor that can show hex (NotePad++ or Sublime are good) and find that extra character.

A lazy, risky option might be to add a preprocessor to the channel to drop the 0x11 character from the raw incoming message. I do not recommend this.

**Reply by pacmano1** [reply]
Also, if you make the same post on the mirth forum, cross reference please.

**Reply by janjovellanos** [reply]
Thank you both for your response and advice! Apologies, as this makes my first post on here. I'll be sure to adhere to the standards moving forward.
